components/adalo/actions/create-record/create-record.mjs

```diff
@@ -0,0 +1,29 @@
+import base from "../common/base.mjs";
+
+export default {
+  ...base,
+  key: "adalo-create-record",
+  name: "Create Record",
+  description: "Create a new record. [See docs here](https://help.adalo.com/integrations/the-adalo-api/collections)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...base.props,
+    data: {
+      label: "Data",
+      description: "The data to create record. E.g. `{ \"Email\": \"string\", \"Username\": \"string\", \"Full Name\": \"string\" }`",
```

Comment:
I think you can create different UI props for better UX right? `email, username, fullName` and then you can build your `data` object instead of allowing the user to set it